photon-icons/photon-icons.json

```diff
@@ -0,0 +1,36 @@
+{
+  "license": "This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.",
+  "fills": [
+    "blue-40",
+    "blue-50",
+    "context-fill",
+    "green-90",
+    "grey-10-opacity-60",
+    "grey-10-opacity-80",
+    "grey-90-opacity-60",
+    "grey-90-opacity-80",
+    "red-60",
+    "white",
+    "yellow-60"
+  ],
+  "icons": {
+    "accessibility": {
+      "name": "Accessibility",
+      "tags": [
+        "a11y",
+        "accessible",
+        "inclusion",
+        "inclusive"
+      ],
+      "categories": [
+        "developer",
+        "general"
+      ],
+      "source": {
```

Comment:
Should we add the source code or path/to/svg/file?

Reply:
Alternatively as future-friendly proposal we could list the sources like this

```
"source": { // Svg source code or path/to/svg/file?
        "web": {
          "16": "INSERT CODE OR PATH HERE",
          "24": "INSERT CODE OR PATH HERE",
          "32": "INSERT CODE OR PATH HERE"
        },
        "android": {
          "20": "INSERT CODE OR PATH HERE" // Android, bounding-box/viewBox is 24x24
        },
        "ios": {
          "24": "INSERT CODE OR PATH HERE" // iOS
        }
      }
```

Reply:
Path. Also I wonder if we should list the svgs separately, and reference them, since they'll be the same for most cases…

Reply:
Are there different fills for different formats? Like, can we have a `context-fill`ed android png?

Reply:
> Path. Also I wonder if we should list the svgs separately, and reference them, since they'll be the same for most cases…

They look the same but they are not. For web and desktop we draw icons on a 16x16 pixel grid with a 1 and 2px stroke. For Android we draw icons on a 20x20 pixel grid with 1 and 2px stroke. Icons are then placed in a 24x24px bounding box. For iOS we draw icons on a 24x24 pixel grid with 2 and 3px stroke. If we would scale the 16x16 icon to 20 or 24px we would get icons strokes that are not aligned to the pixel grid, so this is why we need separate path files. 

> Are there different fills for different formats? Like, can we have a context-filled android png?

That's a good question, I'll look more into that!